bodhi/server/util.py

```diff
@@ -1087,12 +1088,14 @@ def call_api(api_url, service_name, error_key=None, method='GET', data=None, hea
         time.sleep(1)
         return call_api(api_url, service_name, error_key, method, data, headers, retries - 1)
     elif rv.status_code == 500:
+        log.debug(rv.text)
```

Comment:
```
bodhi/tests/server/test_util.py:191
AttributeError: 'FakeResponse' object has no attribute 'text'
```
It seems our FakeResponse object doesn't have a `text' property:
https://github.com/fedora-infra/bodhi/blob/51e8d2ed3c3410d5052042069cedac95391b4900/bodhi/tests/server/test_util.py#L181-L186